addons/editor-dark-mode/experimental_editor.css

```diff
@@ -210,3 +210,6 @@ img[src="/static/assets/63e5827c1506216bd7c9927a4e5eb558.svg"] {
   fill: #282828;
   stroke: #444;
 }
+.monitor_label_ci1ok {
+  color: #fff !important;
```

Comment:
Is `!important` required?

Reply:
yeah website darkmode is over editor darkmode.

Reply:
> Is `!important` required?

Probably (to override website dark mode).

Reply:
Looks like it is because it conflicts with dark-www, which needs to manually set back the monitor color to gray, bc Scratch forgot to set a color for it so it inherits it from the body.